app/views/suites/merge.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= provide(:title, "Merge suite - " + @suite.number + " & " + @other_suite.number) %>
```

Comment:
How about `"Merge suites #{@suite.number} and #{@other_suite.number}"`? You can update the heading as well.